Feature/tildeequal support

Cielquan opened the pull request:
I like to configure the version in setup.cfg like `~=3.6` instead of `>=3.6, <4`.

I added support with version definition with '~='. I also added a corresponding test and updated the README.

asottile commented:
why would you exclude python 4 before you know it's broken?  you're just creating potentially unnecessary pain for your users in the future :(

asottile commented on flake8_typing_imports.py:
```diff
@@ -556,7 +556,7 @@ def parse_options(cls, options: Any) -> None:
         for part in cfg['options']['python_requires'].split(','):
             part = part.strip()
-            if part.startswith('>='):
+            if part.startswith('>=') or part.startswith('~='):
```
`~=` is not the same as `>=` so this isn't correct -- `~=` can't correctly be represented in this scheme which depends on *minimum* version only, there is intentionally no upper bound

for example `~=3.6.4` would not match 3.7.0

Cielquan commented:
Sry for the late reply .. somehow the mail notifications went all to spam.
___

> `~=` is not the same as `>=` so this isn't correct -- `~=` can't correctly be represented in this scheme which depends on _minimum_ version only, there is intentionally no upper bound
> 
> for example `~=3.6.4` would not match 3.7.0

I read PEP 440 and now understand that I misunderstood the `~=` version operator. I thought the following:
```
~=3.8 equal to  >=3.8, <4 (correct)
~=3.8.1 equal to >=3.8.1, <4 (wrong)
```
___

> why would you exclude python 4 before you know it's broken? you're just creating potentially unnecessary pain for your users in the future :(

I wanted to make sure the code works before adding Py4. But you have a point. If there is a bug it can be issued and fixed like I did with #20. If there is none you can save some time.
Furthermore I added labels for the supported python versions anyway so one can see which shall work.

I will change that thanks :)
___

I will close this PR because it's unnecessary.
